artifactory/commands/docker/pull.go

```diff
@@ -23,6 +24,12 @@ func (dpc *DockerPullCommand) Run() error {
 	if errorutils.CheckError(err) != nil {
 		return err
 	}
+
+	err = docker.IsVersionSupported()
```

Comment:
Any reason to do *docker login* with unsupported docker versions?
The login in that case will fail...

Reply:
Moved to the beginning of the function